Rakefile

```diff
@@ -116,10 +116,10 @@ NCURSES_URL = "http://ftpmirror.gnu.org/ncurses/ncurses-#{NCURSES_VERSION}.tar.g
 READLINE_VERSION = '6.3'
 READLINE_URL = "http://ftpmirror.gnu.org/readline/readline-#{READLINE_VERSION}.tar.gz"
 
-RUBY__VERSION = '2.2.3'
+RUBY__VERSION = '2.2.6'
```

Comment:
Think this was just trying to understand if this was a problem, I don't have a problem with reverting back to that.